packages/sdk/src/sdk/api/generated/default/models/ClaimRewardsResponseDataInner.ts

```diff
@@ -0,0 +1,98 @@
+/* tslint:disable */
+/* eslint-disable */
+// @ts-nocheck
+/**
+ * API
+ * Audius V1 API
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+/**
+ * 
+ * @export
+ * @interface ClaimRewardsResponseDataInner
+ */
+export interface ClaimRewardsResponseDataInner {
+    /**
+     * The challenge ID
+     * @type {string}
+     * @memberof ClaimRewardsResponseDataInner
+     */
+    challengeId?: string;
```

Comment:
Are these actually optional? Or could we update the swagger definition so that things guaranteed to be in the response aren't marked optional here.

Reply:
they are!